xml/admin_saltcluster.xml

```diff
@@ -1180,28 +1191,49 @@ OSD_ID  HOST         STATE                    PG_COUNT  REPLACE  FORCE  STARTED_
     </tip>
    </step>
    <step>
-    <para>
-     On the &smaster;, install the <package>ceph-salt</package> package if it
-     is not already installed (see <xref linkend="deploy-cephadm-cephsalt"/>),
-     and include the new &adm; in the cluster as described in
-     <xref linkend="deploy-cephadm-configure-minions"/> and
-     <xref linkend="deploy-cephadm-configure-admin"/>.
-    </para>
+     <para>
+       Install the <package>salt-master</package> package and, if applicable, the
+       <package>salt-minion</package> package on the new &smaster;.
+     </para>
    </step>
    <step>
-    <para>
-     Import the backed up cluster configuration and apply it:
-    </para>
+     <para>
+      Install &cephsalt; on the new &smaster; node:
+     </para>
 <screen>
-&prompt.smaster;ceph-salt import <replaceable>CLUSTER_CONFIG</replaceable>.json
-&prompt.smaster;ceph-salt apply
+&prompt.smaster;zypper install ceph-salt
+&prompt.smaster;systemctl restart salt-master.service
+&prompt.smaster;salt '*' saltutil.sync_all
 </screen>
-   <important>
+     <important>
+       <para>
+         Make sure to run all three commands before continuing. The commands are
+         idempotent, so it does not matter if they get repeated.
```

Comment:
```suggestion
         idempotent&mdash;it does not matter if they get repeated.
```

Reply:
Just a suggestion, as IMHO this is an obscure word. Removing the "so" means that nobody would feel bad if they did not know what "idempotent" meant.

Reply:
Agreed, good thoughts

Reply:
In the U.S. to remove the "so" we would join the two sentences with a colon, not an em-dash: 

    The commands are idempotent: it does not matter if they get repeated.

(Just a thought...)

Reply:
I leave this to @lproven

Reply:
This is news to me! Saying that, I am absolutely fine with it. I defer to the native speaker of American English. ;-)